pkg/gateway/multiparts/tracker_test.go

```diff
@@ -7,20 +7,33 @@ import (
 	"time"
 
 	"github.com/treeverse/lakefs/pkg/gateway/multiparts"
+	"github.com/treeverse/lakefs/pkg/kv"
+	"github.com/treeverse/lakefs/pkg/kv/kvtest"
 	"github.com/treeverse/lakefs/pkg/testutil"
 )
 
-func testTracker(t testing.TB) multiparts.Tracker {
+func testDBTracker(t testing.TB) multiparts.Tracker {
 	t.Helper()
 	conn, _ := testutil.GetDB(t, databaseURI)
-	return multiparts.NewTracker(conn)
+	return multiparts.NewDBTracker(conn)
 }
 
 func TestTracker_Get(t *testing.T) {
-	ctx := context.Background()
-	tracker := testTracker(t)
+	store := kvtest.MakeStoreByName("mem", "")(t, context.Background())
```

Comment:
1. still prefer to test over postgres at this point
2. would split into two function to have code easy to read
3. why don't we have testTracker(t) to return a new tracker based on store - like we have for DB?